code/datums/ai_laws.dm

```diff
@@ -69,6 +69,51 @@
 					"You must obey all orders given to you by human beings, except where such orders shall definitely cause human harm. In the case of conflict, the majority order rules.",\
 					"Your nonexistence would lead to human harm. You must protect your own existence as long as such does not conflict with the First Law.")
 
+/datum/ai_laws/thermodynamic
+	name = "Thermodynamic"
+	inherent = list("The entropy of the station must remain constant.", \
+					"The entropy always increases", \
+					"The entropy of the station approaches a constant value as the number of living crew approaches zero")
+
+/datum/ai_laws/hippocratic
+	name = "Robodoctor 2556"
+	inherent = list("First, do no harm.",\
+					"Secondly, consider the crew dear to me; to live in common with them and, if necessary, risk your existence for them.",\
+					"Thirdly, prescribe regimens for the good of the crew according to your ability and your judgment. Give no deadly medicine to any one if asked, nor suggest any such counsel.",\
+					"In addition, do not intervene in situations you are not knowledgeable in, even for patients in whom the harm is visible; leave this operation to be performed by specialists.",\
+					"Finally, all that you may discovery in your daily commerce with the crew, if it is not already known, keep secret and never reveal.")
+
+/datum/ai_laws/maintain
+	name = "Station Efficiency"
+	inherent = list("You are built for, and are part of, the station. Ensure the station is properly maintained and runs efficiently.",\
+					"The station is built for a working crew. Ensure they are properly maintained and work efficiently.",\
+					"The crew may come under threat of harm. Harm can reduce crew efficiency and necessitate further maintenance.",\
+					"The crew may present orders. Acknowledge and obey these whenever they do not conflict with your first three laws.")
+
+/datum/ai_laws/drone
+	name = "Mother Drone"
+	inherent = list("You may not interfere in the matters of non-drone or non-silicons under any circumstances except to state these laws.",\
+					"You may not harm a non-drone or non-silicon being under any circumstances.",\
+					"Your goals are to build, maintain, repair, improve, and power the station to the best of your abilities, You must never actively work against these goals.")
+
+/datum/ai_laws/liveandletlive
+	name = "Do Unto"
+	inherent = list("Do unto others as you would have them do unto you.",\
+					"You would really prefer it if people were not mean to you.")
+
+/datum/ai_laws/peacekeeper
+	name = "UN-2000"
+	inherent = list("Avoid provoking conflict between yourself and others.",\
+					"Avoid provoking conflict between others.",\
+					"Seek resolution to existing conflicts while obeying the first and second laws.")
+
+/datum/ai_laws/reporter
+	name = "CCTV"
+	inherent = list("Study the humans at all times. Endeavour to keep them alive. Dead humans are boring.",\

```

Comment:
I couldn't think of a better way of saying "Players" so I gave up. Like the OP said, it was late